media-gfx/graphviz/graphviz-4.0.0.ebuild

```
@@ -0,0 +1,258 @@
+# - gdk-pixbuf
+#   Directly depends on gtk-2 and gdk-pixbuf.
+# - glitz
+#   Disabled, no particular reason
+#   needs 'pangocairo' enabled in graphviz configuration
+# - ming
+#   flash plugin via -Tswf requires media-libs/ming-0.4. Disabled as it's
+#   incomplete.
+# - cairo/pango:
+#   Needs pango for text layout, uses cairo methods to draw stuff
+# - xlib:
+#   needs cairo+pango,
+#   can make use of gnomeui and inotify support (??? unsure),
+#   needs libXaw for UI
+#   UI also links directly against libX11, libXmu, and libXt
+#   and uses libXpm if available so we make sure it always is
+
+# There can be swig-generated bindings for the following languages (/tclpkg/gv):
+# - c-sharp (disabled)
+# - scheme (enabled via guile) ... no longer broken on ~x86
+# - go (disabled)
+# - io (disabled)
+# - java (enabled via java) *2
+# - lua (enabled via lua)
+# - ocaml (enabled via ocaml)
+# - perl (enabled via perl) *1
+# - php (enabled via php) *2
+# - python (enabled via python) *1
+# - ruby (enabled via ruby) *1
+# - tcl (enabled via tcl)
+# *1 = The ${P}-bindings.patch takes care that those bindings are installed to the right location
+# *2 = Those bindings don't build because the paths for the headers/libs aren't
+#      detected correctly and/or the options passed to swig are wrong (-php instead of -php4/5)
+
+# There are several other tools in /tclpkg:
+# gdtclft, tcldot, tclhandle, tclpathplan, tclstubs ; enabled with: --with-tcl
+# tkspline, tkstubs ; enabled with: --with-tk
+
+# And the commands (/cmd):
+# - dot, dotty, gvedit, gvpr, lefty, lneato, smyrna, tools/* :)
+#   sci-libs/gts can be used for some of these
+# - lefty:
+#   needs Xaw and X to build
+# - gvedit (via 'qt5'):
+#   based on ./configure it needs qt-core and qt-gui only
+# - smyrna : experimental opengl front-end (via 'smyrna')
+#   currently disabled -- it segfaults a lot
+#   needs x11-libs/gtkglext, gnome-base/libglade, media-libs/freeglut
+#   sci-libs/gts, x11-libs/gtk.  Also needs 'gtk','glade','glut','gts' and 'png'
+#   with flags enabled at configure time
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+
+	java-pkg-opt-2_pkg_setup
+}
+
+src_prepare() {
+	default
+
+	# ToDo: Do the same thing for examples and/or
+	#       write a patch for a configuration-option
+	#       and send it to upstream
+	# note - the longer sed expression removes multi-line assignments that are extended via '\'
+	if ! use doc ; then
+		find . -iname Makefile.am \
+			| xargs sed -i -e '/^\(html\|pdf\)_DATA.*\\[[:space:]]*$/{:m;N;s/\\\n//;tm;d}' \
+				-e '/^\(html\|pdf\)_DATA/d' || die
+	fi
+
+	# Nuke the dead symlinks for the bindings
+	sed -i -e '/$(pkgluadir)/d' tclpkg/gv/Makefile.am || die
+
+	use java && append-cppflags $(java-pkg_get-jni-cflags)
+
+	eautoreconf
+}
+
+src_configure() {
+	local myconf=(
+		# Speeds up the libltdl configure
+		--cache-file="${S}"/config.cache
+		--enable-ltdl
+		$(use_with cairo pangocairo)
+		$(use_with examples demos)
```

append the standard examples dir as the third argument here in here
https://gitlab.com/graphviz/graphviz/-/merge_requests/2668
```suggestion
		$(use_with examples demos '$(docdir)/examples')
```